Formula/nginx-full.rb

```diff
@@ -246,9 +247,14 @@ def install
       args << "--add-module=#{HOMEBREW_PREFIX}/share/ngx-devel-kit"
     end
 
+    # Njs module configuration path needs to be appended with "/nginx" compared to other modules
+    if build.with?("njs-module")
+      args << "--add-module=#{HOMEBREW_PREFIX}/share/njs-nginx-module/nginx"
+    end
+
     # Third Party Modules
     self.class.third_party_modules.each_key do |name|
-      if build.with? "#{name}-module"
+      if build.with? "#{name}-module" && name != "njs"
```

Comment:
I could not it to work - combined with the `(share+"njs-nginx-module").install Dir["nginx/*"]`. The main reason is that the njs module does see the nginx module as a configuration only, and the rest of the code is in another folder ("../nxt" and "../njs").